3/ancestor.deps

```diff
@@ -0,0 +1 @@
+../2.10/ansible-2.10.0.deps
```

Comment:
The name of this file is random. Antsibull goes through all `.deps` files and reads them all; and identifies this dependency file as the ancestor by comparing its version with the ancestor version from `changelog.yaml`. Naming the symlink this way should make it more obvious though.